examples/generator-fullstack/.graphqlrc.yml

```diff
@@ -1,6 +1,4 @@
-schema:
-    ./server/src/schema/index.ts:
-        require: ts-node/register
+schema: './model/*.graphql'
```

Comment:
GraphQL config is not only for graphback. There will be other projects basing on the entire generated schema. 
We cannot do that as it will break everything

Reply:
Sure thing. I've changed it now to load schema from directory (the old way).